packages/hdwallet-ledger-webusb/src/transport.ts

```diff
@@ -48,13 +48,22 @@ export async function getTransport(): Promise<TransportWebUSB> {
 
   let ledgerTransport
```

Comment:
Suggestion:

```
  try {
    let transport = await TransportWebUSB.openConnected()
    if (transport) {
      return transport
    }

    // Use create() rather than request() to avoid the WebUSB pairing dialog when possible.
    return await TransportWebUSB.create()
  } catch (err) {
    if (err.name === 'TransportInterfaceNotAvailable') {
      throw new ConflictingApp('Ledger')
    }
    throw new WebUSBCouldNotPair('Ledger', err.message)
  }
```

Reply:
@keepkeyjon Part of the initial problems was that if `openConnected` threw an error, it would short circuit the logic to the `catch` block, resulting in `create/request` never being called as a fallback. See my comment below though in regards to `create` functionality which should make this all much cleaner anyways.